File: apps/pwa/src/api/remove_music_from_musicbill.ts
import { request, Method } from '.';

function removeMusicFromMusicbill(musicbillId: string, musicId: string) {
  return request({
    method: Method.DELETE,
    path: '/api/musicbill_music',
    params: { musicbillId, musicId },
  });
}

export default removeMusicFromMusicbill;

File: apps/flutter/lib/server/api/remove_music_from_musicbill.dart
import '../request.dart';

Future<void> removeMusicFromMusicbill({
  required String musicbillId,
  required String musicId,
}) async {
  await httpDelete(
    path: "/api/musicbill_music",
    query: {"musicbillId": musicbillId, "musicId": musicId},
    withToken: true,
  );
}
